mimic/test/test_maas.py

```diff
@@ -382,11 +382,20 @@ def test_agenthostinfo(self):
         """
         fetch agent host info
         """
-        req = request(self, self.root, "GET", self.uri + '/views/agent_host_info', '')
+        ecan = self.get_ecan_object_ids()
```

Comment:
use `self.entity_id` in this test. If this is changed across all tests for all the resource ids, it will fix #156